examples/api/test/material/menu_anchor/menu_anchor.2_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/menu_anchor/menu_anchor.2.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The menu should display 3 items', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.MenuAnchorApp(),
+    );
+
+    expect(find.widgetWithText(AppBar, 'MenuAnchorButton'), findsOne);
+    expect(
+      find.descendant(of: find.byType(MenuAnchor), matching: find.widgetWithIcon(IconButton, Icons.more_horiz)),
+      findsOne,
+    );
+
+    await tester.tap(find.byIcon(Icons.more_horiz));
+    await tester.pumpAndSettle();
+
+    for (int i = 1; i <= 3; i++) {
+      expect(find.widgetWithText(MenuItemButton, 'Item $i'), findsOne);
+    }
+  });
+
```

Comment:
Consider removing this empty line

Reply:
Argh!! Always this empty line 😱 😞 

I removed it in
[test: Remove empty line](https://github.com/flutter/flutter/pull/151399/commits/7fa738679b0ca0c8754291a3b1e87e9804265247)